Common/src/config_structure.cpp

```diff
@@ -1562,16 +1562,18 @@ void CConfig::SetConfig_Options(unsigned short val_iZone, unsigned short val_nZo
   addDoubleOption("DEFORM_TOL_FACTOR", Deform_Tol_Factor, 1E-6);
   /* DESCRIPTION: Deform coefficient (-1.0 to 0.5) */
   addDoubleOption("DEFORM_COEFF", Deform_Coeff, 1E6);
-  /* DESCRIPTION: Type of element stiffness imposed for FEA mesh deformation (INVERSE_VOLUME, DEF_WALL_DISTANCE, WALL_DISTANCE, CONSTANT_STIFFNESS) */
-  addEnumOption("DEFORM_STIFFNESS_TYPE", Deform_Stiffness_Type, Deform_Stiffness_Map, BOUNDARY_DISTANCE);
+  /* DESCRIPTION: Deform limit in m or inches */
+  addDoubleOption("DEFORM_LIMIT", Deform_Limit, 1E6);
```

Comment:
When is this new option for the deform limit useful? Is this what you meant by the 3rd bullet in your PR description, basically controlling the max value that will be used when far from a solid wall (to make the convergence behave better)?

Reply:
I found some particular problems where it was useful to limit the extension of the deformation. In fact, there was something similar in the solver (you can define a box and the solver only will move the points  that are inside the box). I found the box system a little bit complicated to define and I thought that something based on the distance to the surface was easier to understand... e.g. DEFORM_LIMIT=1 only moves the points that are 1m away from the solid surface.